### ISSUE TITLE
Problems with exam

### ISSUE THREAD

**Issue by ltropp** [upjs-jf/mdh#19]
Hi @mvala ,

I have tried run examTask after source env-osx.sh but it crashed.
I attach my terminal output.
I am looking forward for your reply.
Thank you.

Lukas

PS: it is necessary to make ./make.sh install after I change anything in MdhTaskInput or MdhTaskBin ?

[TerminalOutputMATO.txt](https://github.com/upjs-jf/mdh/files/1159048/TerminalOutputMATO.txt)

**Comment by mvala** [comment]
you have to do it from root directory of project and run
```
scripts/make.sh clean install
```

**Comment by mvala** [comment]
Yes it is nessesery to recompile and install after you change something in classes (Mdh dir). When you change macro you don't have to recompile
